### ISSUE TITLE
Failed to connect doh-server via dnscrypt-proxy

### ISSUE THREAD

**Issue by ookangzheng** [m13253/dns-over-https#14]
Env: MacOSX High sierra
sdns: https://dnscrypt.info/stamps/
dnscrpyt-proxy: https://github.com/jedisct1/dnscrypt-proxy/wiki/Installation-macOS

dnscrypt-proxy config.toml
```
// server names
server_names = ['blahdns']
// static part
[static]
  [static.'blahdns']
  stamp = 'sdns://AgEAAAAAAAAAAAALZG9oLmRhdHQucHcKL2Rucy1xdWVyeQ'
```
when I dig google.com @127.0.0.1, not respond

**Comment by m13253** [comment]
Thank you for your report.

I am aware of this issue.
Since there was no former experience of using DoH with DNScrypt, I will investigate it and reply you later.

**Comment by m13253** [comment]
It is a bug of DNSCrypt-Proxy.
But I fixed it through commit 523b6120b9a573237e3ab6d1e71515915ea5837a.

Remember to report a bug at their side.

**Comment by m13253** [comment]
By the way, comments should begin with `#`, not `//`.
